chore: add close button to troubleshooting and help page

jannikbertram opened the pull request:
### What does this PR do?

- Add an X button to pages which don't have breadcrumbs

### Screenshot/screencast of this PR

<img width="682" alt="image" src="https://github.com/containers/podman-desktop/assets/17082165/5389673f-51de-4f79-9d7a-e52d5c0c614a">

<img width="684" alt="image" src="https://github.com/containers/podman-desktop/assets/17082165/33c29e2d-f45d-43e3-90c7-485b135e9ca0">

### What issues does this PR fix or reference?
Closes #2878 

### How to test this PR?

Visit the "Troubleshooting" and "Help" page

deboer-tim commented:
@mairin The original design was no breadcrumbs on these two pages, but it sounds like it's throwing people off. Thoughts?

@jannikbertram If we do go down this route we should remove the property entirely instead of setting it true everywhere, and likely rename Home to Dashboard since I assume that's where it goes 'back' to.

jannikbertram commented:
Another option would be to only include the ❌ icon and remove the breadcrumbs. 
I just thought I propose this obvious approach first. 
Just let me know which design is preferred

mairin commented:
@deboer-tim @jannikbertram Help and Troubleshooting didn't have breadcrumbs because they sort of belong to the status bar... Neither has a presence on the dashboard (not that they shouldn't, they just don't) so semantically linking back to the Dashboard as a parent in a breadcrumb feels weird.

I do think the X in the upper right is a good idea and may have even been in the original design. I really like how you have it going to the last previously-viewed screen when the user hits the X.

I would say let's try a light touch first and just do the X, and try to side step the 🤯-ness of the breadcrumbs for now?

jannikbertram commented:
Thanks for clarifying this @mairin. I added a new commit and updated the PR description.

This is how it looks when you use the X button together with actions and breadcrumbs:

<img width="687" alt="image" src="https://github.com/containers/podman-desktop/assets/17082165/77eabfd0-be65-464d-8876-48083d4e9ed7">

I'm not sure if it should be possible like this. I feel like this really is about small details, so please let me know if you want this to behave differently. I can keep updating this until we find a consistent solution :)

deboer-tim commented:
That looks odd to me. I don't think this PR should change the location of the close button on all the other pages (it should remain on the 'breadcrumb bar'), only add a close button to the ones that don't have breadcrumbs.

jannikbertram commented:
I updated the PR again

> only add a close button to the ones that don't have breadcrumbs.

now it simply does this ☝️

mairin commented:
@jannikbertram this looks great to me, just tested it out :)

jeffmaury reviewed (changes requested):
Fix unit tests

jannikbertram commented:
@jeffmaury can you trigger the unit tests again? They should be working now

deboer-tim reviewed (approved):
LGTM. @jeffmaury, assuming you're 👍🏼 now.

jeffmaury reviewed (approved):
LGTM. Thanks @jannikbertram for this
